apps/labs/posts/but-what-are-traitlets.md

```diff
@@ -0,0 +1,351 @@
+---
+title: 'Traitlets - an introduction & use in Jupyter configuration management'
+published: August 30, 2020
+author: matthias-bussonnier
+description: 'You have probably seen Traitlets in applications, you likely even use it.
+The package has nearly 5 million downloads on conda-forge alone. But, what is Traitlets?
+In this post we'll answer this question along with where Traitlets came from,
```

Comment:
```suggestion
In this post we''ll answer this question along with where Traitlets came from,
```
Address Vercel build error `error YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 9, column 9`, updated single quotes to properly close YAML description